### PR TITLE
Setup metric-related classes

### PR DESCRIPTION
**Pull request by Natchica**

### CODE REVIEW

File: src/main/java/com/iexec/standalone/metric/MetricService.java [hunk 1]
```diff
@@ -1,61 +0,0 @@
...
-import com.iexec.standalone.task.TaskStatus;
-import com.iexec.standalone.task.event.TaskStatusesCountUpdatedEvent;
-import com.iexec.standalone.worker.WorkerService;
-import org.springframework.context.event.EventListener;
-import org.springframework.stereotype.Service;
-
-import java.util.LinkedHashMap;
-
-@Service
-public class MetricService {
```
Comment (jbern0rd) [inline src/main/java/com/iexec/standalone/metric/MetricService.java:29]: I think this removal is a bit too strong.
We may be interested to know if the standalone worker is full or not and follow the number of tasks per status on the `/metrics` endpoint ?
Do we want to remove this or try to keep this and we need to study how to modify this class ?
If something has already been decided regarding this, I will follow the decision

File: src/test/java/com/iexec/standalone/metric/MetricsServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    }
+    // endregion
+
+    // region getCompleteComputeDurations
+    @Test
+    void shouldGetCompleteComputeDurations() {
+        when(appComputeDurationsService.getChainTaskIds()).thenReturn(List.of(CHAIN_TASK_ID_1, CHAIN_TASK_ID_2));
+
+        // First task has completed
+        System.out.println("Task 1 :" + CHAIN_TASK_ID_1);
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/metric/MetricsServiceTests.java]: minor but `System.out.println` calls should be avoided, even in tests

Comment (Natchica) [reply src/test/java/com/iexec/standalone/metric/MetricsServiceTests.java]: Forgot to delete them after debugging

**Review by jbern0rd — APPROVED** [review]
LGTM :rocket:

**Pull request merged**